Tao_Current.txt

```diff
@@ -163,133 +187,159 @@ The IETF discussion list is unmoderated. This means that all can express their o
 
 For many people, IETF meetings are a breath of fresh air when compared to the standard computer industry conferences. There is no exposition hall, few tutorials, and no big-name industry pundits. Instead, there is lots of work, as well as a fair amount of time for socializing for many participants. IETF meetings are of little interest to sales and marketing folks, but of high interest to engineers and developers.
 
-The general flow of an IETF meeting is that it begins with tutorials and an informal gathering on Sunday, and that there are WG and BoF meetings Monday through Friday. WG meetings last for between 1 and 2.5 hours each, and some WGs have meetings multiple times during the week, depending on how much work they anticipate doing.
+The general flow of an IETF meeting is that it begins with a hackathon on Saturday and Sunday, tutorials and an informal gathering on Sunday, and that there are WG and BoF meetings Monday through Friday. WG meetings last for between 1 and 2.5 hours each, and some WGs have meetings multiple times during the week, depending on how much work they anticipate doing.
+
+There are two plenary sessions, one technical and one administrative, in the evenings during the week (mostly Wednesday). The technical plenary is organized by the IAB and usually has one or two panels of experts on topics of interest across many WGs and Areas. The administrative plenary, organized by the IETF Chair, covers things like progress reports from the RFC Editor and announcements of upcoming meetings. The plenaries are a good time to share with the IESG and IAOC. Praise is welcome, but more often concerns and gripes are raised. In recent meetings, the two plenaries have been combined.
+
 
-There are two plenary sessions, one technical and one administrative, in the evenings during the week. The technical plenary is organized by the IAB and usually has one or two panels of experts on topics of interest across many WGs and Areas. The administrative plenary, organized by the IETF Chair, covers things like progress reports from the RFC Editor and announcements of upcoming meetings. The plenaries are a good time to share with the IESG and IAOC. Praise is welcome, but more often concerns and gripes are raised.
+Currently, the IETF meets in North America, Europe, and Asia, approximately once a year in each region. The past few meetings have had about 1,200 attendees. There have been more than 100 IETF meetings so far, and a list of upcoming meetings is available on the IETF web pages, [https://www.ietf.org/how/meetings/upcoming]. You can read more about the meeting selection process and criteria here ([https://datatracker.ietf.org/doc/draft-ietf-mtgvenue-meeting-policy/] [https://datatracker.ietf.org/doc/draft-ietf-mtgvenue-iaoc-venue-selection-process/])
```

Comment:
The attendance stats on the past few meetings are not correct. I would suggest dropping this since it will be instantly out-of-date.

Reply:
Removed: The past few meetings have had about 1,200 attendees.